app.go

```diff
@@ -348,8 +348,9 @@ type Config struct {
 	// Read EnableTrustedProxyCheck doc.
 	//
 	// Default: []string
-	TrustedProxies    []string `json:"trusted_proxies"`
-	trustedProxiesMap map[string]struct{}
+	TrustedProxies        []string `json:"trusted_proxies"`
+	trustedProxiesMap     map[string]struct{}
```

Comment:
@ReneWerner87 do you know the reasoning for using `map[string]struct{}` over `[]string` here?

Reply:
not really, we should ask the creator 

https://github.com/gofiber/fiber/pull/1397/files
@oroshnivskyy

Reply:
To decrease the time complexity of IsProxyTrusted function.